cgcnn2/data.py

```
@@ -22,33 +22,20 @@ def collate_pool(dataset_list):
-          - atom_fea (torch.Tensor): shape (n_i, atom_fea_len)
-            Atom features for each atom in the crystal
-          - nbr_fea (torch.Tensor): shape (n_i, M, nbr_fea_len)
-            Bond features for each atom's M neighbors
-          - nbr_fea_idx (torch.LongTensor): shape (n_i, M)
-            Indices of M neighbors of each atom
-          - target (torch.Tensor): shape (1, )
-            Target value for prediction
-          - cif_id (str or int)
-            Unique ID for the crystal
+        dataset_list (list of tuples): List of tuples for each data point. Each tuple contains:
+        atom_fea (torch.Tensor): shape (n_i, atom_fea_len) Atom features for each atom in the crystal
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)